pkg/log/zap/zap.go

```diff
@@ -223,10 +252,18 @@ func (o *Options) BindFlags(fs *flag.FlagSet) {
 		"Development Mode defaults(encoder=consoleEncoder,logLevel=Debug,stackTraceLevel=Warn). "+
 			"Production Mode defaults(encoder=jsonEncoder,logLevel=Info,stackTraceLevel=Error)")
 
+	// Set TimeEncoder value
+	var timeEncVal timeEncoderFlag
+	timeEncVal.setFunc = func(fromFlag zapcore.TimeEncoder) {
+		// Set o.EncoderConfigOptions with zap-time-encoder
+		TimeEncoder(fromFlag)(o)
+	}
+	fs.Var(&timeEncVal, "zap-time-encoder", "Zap time encoding format, defaults to 'epoch'")
```

Comment:
`--zap-time-encoder` and `--zap-encoder` are mutually exclusive right? Can you update the descriptions to include this info?

Reply:
They are not designed to be mutually exclusive, `--zap-encoder` takes either console/json, and if `--zap-time-encoder` is given, then will set the Encoder with given time format.
Example, "--zap-encoder=json --zap-time-encoder=millis" will result in JSON encoder with `EpochMillisEncoder` set.